src/plugin/index.ts

```diff
@@ -50,11 +51,10 @@ export default function nextTranslate(nextConfig: any = {}) {
     ...nextConfig,
     i18n: {
       ...i18n,
-      ...restI18n,
```

Comment:
this should not be deleted. What should be done is to remove the restI18n properties that are next-translate properties and not i18n properties. That is, make sure that everything in restI18n are properties of i18n's Next.js.

Reply:
> this should not be deleted. What should be done is to remove the restI18n properties that are next-translate properties and not i18n properties. That is, make sure that everything in restI18n are properties of i18n's Next.js.

`restI18n` has no properties that are valid for Next.js.

![image](https://user-images.githubusercontent.com/28112929/184834394-27da9d54-1501-4290-8cb3-e7be96925158.png)

Reply:
Many people use the `i18n.js` file to put all the features of Next-translate and Next.js, to have it together in one place. The thing is that here we should remove the Next-translate ones so that only the Next.js ones are passed.

Reply:
I see now. Never realised that next.js properties could be declared in next-translate, even though the plugin has no use for them. I'll update the PR soon enough.